Snowflake Privatelink VPC Endpoint ID Docs (#5268)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/secure/snowflake-privatelink.md

3. Select the private endpoint from the dropdown (this will automatically populate the hostname/account field).
4. Configure the remaining data platform details.
5. Test your connection and save it.

## Configuring Network Policies

> If your organization uses [Snowflake Network Policies](https://docs.snowflake.com/en/user-guide/network-policies) to restrict access to your Snowflake account, you will also need to add a network rule for dbt Cloud. 

Reviewer: ```suggestion
If your organization uses [Snowflake Network Policies](https://docs.snowflake.com/en/user-guide/network-policies) to restrict access to your Snowflake account, you will need to add a network rule for dbt Cloud. 
```

> You may request the VPCE ID from dbt Cloud Support that you can use to create a network policy. 

Reviewer: ```suggestion
You can request the VPCE ID from [dbt Cloud Support](mailto:support@getdbt.com), that you can use to create a network policy. 
```

### Using the UI

Open the Snowflake UI and take the following steps:
1. Go to the **Security** tab.
2. Click on **Network Rules**.
3. Click on **Add Rule**.
4. Give the rule a name.
5. Select a database and schema where the rule will be stored. These selections are for permission settings and organizational purposes; they do not affect the rule itself.
6. Set the type to `AWS VPCE ID` and the mode to `Ingress`.

> 7. Type the VPCE ID provided by dbt Cloud Support into identifier box and press **Enter**.

Reviewer: ```suggestion
7. Type the VPCE ID provided by dbt Cloud Support into the identifier box and press **Enter**.
```

8. Click **Create Network Rule**.

<Lightbox src="/img/docs/dbt-cloud/snowflakeprivatelink2.png" title="Create Network Rule"/>

> 9. In the **Network Policy** tab, edit the policy you want to add the rule to.

Reviewer: ```suggestion
9. In the **Network Policy** tab, edit the policy you want to add the rule to. This could be your account-level policy or a policy specific to the users connecting from dbt Cloud.
```

> - _*This could be your account level policy, or a policy specific to the users who will be connecting from dbt Cloud._

Reviewer: ```suggestion
```
